micronaut-monitoring/src/main/java/com/wengnerits/monitoring/micronaut/controller/MainController.java

```diff
@@ -17,22 +17,18 @@
 
 package com.wengnerits.monitoring.micronaut.controller;
 
-
 import com.wengnerits.monitoring.micronaut.service.HelloService;
 import io.micrometer.core.instrument.Counter;
-import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.prometheus.PrometheusMeterRegistry;
 import io.micronaut.http.MediaType;
 import io.micronaut.http.annotation.Controller;
 import io.micronaut.http.annotation.Get;
 import io.micronaut.http.annotation.PathVariable;
 import io.prometheus.client.exporter.common.TextFormat;
-import jakarta.inject.Inject;
-
 import java.util.HashMap;
 import java.util.Map;
 
-@Controller("/")
```

Comment:
is possible to revert this changes in "micronaut-monitoring" module as it's not a part for the "micronaut-kotlin-monitoring" ?